app/Http/Filters/NewOrderFilter.php

```diff
@@ -275,4 +276,9 @@ public function financed_by(string $financed_by)
     {
         $this->builder->where('financed_by', $financed_by);
     }
+    public function bnplOrders(bool $isBNPL = true ){
```

Comment:
Can we have a scenario where $isBNPL could be false?

Reply:
No. i should remove the condition